src/tutorials/jwst-image-scale-bar.md

```diff
@@ -0,0 +1,208 @@
+We can make the image pop a little more by adjusting the display limits, or `clims`. These can be either a tuple of min and max values (i.e. `(0.0, 100.0)`) or a function to calculate them for us based on some criteria. We'll adjust the limits to remove the top and bottom 1% of all  pixels:
+
+```julia
+imview(carina; clims=Percent(98))
+```
+![Carina nebula displayed as an image with higher contrast](../assets/tutorials/jwst-1/carina-2.png)
+
+
+In Julia, [keyword arguments](https://docs.julialang.org/en/v1/manual/functions/#Keyword-Arguments) come after positional arguments and they can be separated by a semi-colon.
+
+
+## Adding World Coordinates
+The image header already contains WCS keys to specify the location of the image in the sky. Let's take a peek at one of these header keys:
+
+```julia
+println(carina["CTYPE1", Comment])
+println(carina["CTYPE1"])
+```
+```
+first axis coordinate type
+RA---TAN
+```
+
+This shows that the coordinate type for the first axis is "RA---TAN".
+
+Now, we'll combine AstroImages with the [Plots.jl](https://docs.juliaplots.org/latest/) library to display these coordinates.
+
+
+```julia
+using Plots
+
+implot(carina; clims=Percent(98))
+```
+![Carina nebula displayed in a plot](../assets/tutorials/jwst-1/carina-3.svg)
+
+
+The image is tilted compared to the right-ascension and declination celestial coordinate system, so the grid is rotated.
+
+The colourbar on the right displays the units of the image: Mega-Janskys per steradian: units of energy per collecting area, per spectral bandwidth, per solid-angle on the sky.
+
+Let's turn off the grid for now:
+```julia
+implot(carina; grid=false, clims=Percent(98))
+```
+![Carina nebula displayed in a plot without grid lines](../assets/tutorials/jwst-1/carina-4.svg)
+
+
+Full documentation for the `implot` function is available in the [AstroImages.jl docs](http://juliaastro.org/AstroImages.jl/dev/api/#AstroImages.implot).
+
+
+## Adding a Scalebar
+
+Another way we can communicate the scale of this image is by adding a scalebar. We'll need to compute the length of a 2D vector (i.e. using the Pythagorean theorem) but we use the LinearAlgebra standard library function `norm` here for convenience.
+
+```julia
+using LinearAlgebra
+
+# Start the scalebar here in pixel coordinates
+start_coord_pix = [1000, 100]
+
+# Convert to world coordinates
+start_coord_world = pix_to_world(carina_full, start_coord_pix)
+
+# Extend by 10 arcminutes of declination
+stop_coord_world  = start_coord_world .+ [0, 1/60]
+
+# Convert back to pixel coordinates
+stop_coord_pix = world_to_pix(carina_full, stop_coord_world)
+
+# Measure how long it is here in pixel coordinates (note: in theory this depends on where in the image we make this calculation because the coordinate system is warped)
```

Comment:
comments here and a couple other spots are too long, causing the rendered code to have a horizontal scroll bar. For the comments, try a hard wrap of 80 or 92 characters.